main.js

```diff
@@ -160,4 +160,217 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+//1a
+const tvTypes = inventory.map((inventory) => {
+  return inventory.name;
+});
+console.log(tvTypes);
+
+
+
+//1b
+const soldOut = inventory.filter((inventory) => {
+  return inventory.sold === inventory.originalStock;
+})
+console.log(soldOut);
+
+
+
+
+//1c
+
+const hasAmbilight = inventory.filter((inventory) => {
+  return inventory.options.ambiLight;
+})
+console.log(hasAmbilight);
+
+
+
+//1d
+inventory.sort((a, b) => a.price - b.price);
+console.log(inventory);
+
+
+
+//2a
+let amountSold = 0;
+for (let i = 0; i < inventory.length; i++) {
+  amountSold += inventory[i].sold;
+}
+console.log(amountSold);
+
+
+
+//2b
+document.getElementById(verkocht);
+verkocht.textContent = amountSold;
+verkocht.style.color = "green";
+
+
+
+//2c
+let amountBought = 0;
+for (let i = 0; i < inventory.length; i++) {
+  amountBought += inventory[i].originalStock;
+}
+console.log(amountBought);
+
+
+
+//2d
+document.getElementById(gekocht);
+gekocht.textContent = amountBought;
+gekocht.style.color = "blue";
+
+
+
+//2e
+let amountAvailable = 0;
+amountAvailable = amountBought - amountSold;
+console.log(amountAvailable);
+
+document.getElementById(opVoorraad);
+opVoorraad.textContent = amountAvailable;
+opVoorraad.style.color = "red";
+
+
+
+//3a en b
+
+function tvBrands(arr) {
+  const tvBrand = arr.map((inventory) =>{
+    return inventory.brand;
+  });
+  const listOfBrands = document.createDocumentFragment();
+  for (let Brand of tvBrand) {
+    const li = document.createElement('li')
+    li.textContent = Brand;
+    listOfBrands.appendChild(li);
+  }
+  const container = document.getElementById('BrandList');
+  container.appendChild(listOfBrands);
+}
+
+tvBrands(inventory);
+
+
+
+
+
+
+//4a
+function tvNames(tv) {
+    return tv.brand + ' ' + tv.type + ' - ' + tv.name;
```

Comment:
Wellicht is dit de volgende keer makkelijker, en ook om te lezen
```suggestion
    return `${tv.brand} ${tv.type} - ${tv.name}``;
```